external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/CachedKafkaProducer.scala

```diff
@@ -38,7 +38,7 @@ private[kafka010] object CachedKafkaProducer extends Logging {
 
   private lazy val cacheExpireTimeout: Long =
     Option(SparkEnv.get).map(_.conf.getTimeAsMs(
-      "spark.kafka.producer.cache.timeout",
+      PRODUCER_CACHE_TIMEOUT.key,
       s"${defaultCacheExpireTimeout}ms")).getOrElse(defaultCacheExpireTimeout)
```

Comment:
No need for the getOrElse if the config has a default, and the default can be removed. I think you need to call the 'get' method with the config object, rather than its key

Reply:
> No need for the getOrElse if the config has a default, and the default can be removed.

`SparkEnv.get` can be `None` and such case the mentioned else handles the situation.

> I think you need to call the 'get' method with the config object, rather than its key

True.